api/metrics/src/main/java/io/opentelemetry/api/metrics/BoundDoubleHistogram.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.api.metrics;
+
+import io.opentelemetry.context.Context;
+import javax.annotation.concurrent.ThreadSafe;
+
+/** A hisgogram instrument that records {@code long} values with preassociated attributes. */
+@ThreadSafe
+public interface BoundDoubleHistogram {
+  /**
+   * Record a value with a pre-bound set of attributes.
+   *
+   * @param value The amount of the measurement.
+   * @param context The explicit context to associate with this measurement.
+   */
+  public void record(double value, Context context);
```

Comment:
for when you're ready... interface methods inherit the access of the class, so shouldn't have additional modifiers on the methods.